contracts/Strategy.sol

```diff
@@ -916,18 +1015,96 @@ contract Strategy is BaseStrategy {
         );
     }
 
+    function _exchangeUsingCrvMIM(
+        uint256 _amount,
+        int128 tokenA,
+        int128 tokenB
+    ) internal {
+        if (_amount == 0 || tokenA == tokenB) {
+            return;
+        }
+
+        crvMIM.exchange_underlying(
+            tokenA,
+            tokenB,
+            _amount,
+            0
```

Comment:
@newmickymousse when exchanging stablecoins I usually like to set `min_dy` to a maximum allowed slippage. Poolpi will say it does not matter because it is flashloanable, but it gives me peace of mind in case the tx ends up in an uncle block. Just sharing so you can make your decision :P

Reply:
I'm adding a `min_dy`